frontend/public/components/nav/admin-nav.tsx

```diff
@@ -161,7 +163,10 @@ const AdminNav = () => (
       <ResourceNSLink resource="rolebindings" name="Role Bindings" startsWith={rolebindingsStartsWith} />
       <ResourceNSLink resource="resourcequotas" name="Resource Quotas" startsWith={quotaStartsWith} />
       <ResourceNSLink resource="limitranges" name="Limit Ranges" />
-      <ResourceNSLink resource={referenceForModel(ChargebackReportModel)} name="Chargeback" required={FLAGS.CHARGEBACK} />
+      <HrefLink href={formatNamespacedRouteForResource(referenceForModel(ChargebackReportModel), 'openshift-metering')}
+                name="Chargeback"
+                required={[FLAGS.CHARGEBACK, FLAGS.CAN_LIST_CHARGEBACK_REPORTS]}
+                startsWith={meteringStartsWith} />
```

Comment:
Does this work? I don't think the path actually starts with `metering.openshift.io`

Reply:
After drilling down into a Reports detail page, so I believe `startsWith` is working.
Chargeback Report listing URL is:
https://console-openshift-console.apps.mydevaws6.devcluster.openshift.com/k8s/ns/openshift-metering/metering.openshift.io~v1alpha1~Report

Reply:
I did wonder about that because all other `startsWith` and urls seem to be much cleaner, such as "clusterroles", "monitoring/alerts", etc..  Wasn't sure if was required for proxy settings?